test/pamela/HTN/statements.main.tpn.edn

```diff
@@ -1,15 +1,6 @@
 {:act-14
- {:args
-  [{:type :equal,
-    :args
-    [{:type :state-variable, :name foo}
-     {:type :literal, :value :illuminated}]}],
-  :argsmap
-  {"condition"
-   {:args
-    [{:type :state-variable, :name foo}
-     {:type :literal, :value :illuminated}],
-    :type :equal}},
+ {:args [(= foo :illuminated)],
```

Comment:
We've lost some important information with this PR.  

- All of the structure of the condition has been lost.  E.g., because the `=` hasn't been expanded into a parsed structure, the consumer needs to do the parsing/expansion of the operand and arguments (that were previously done when creating the IR) again.